### PR TITLE
LPS-111253 Usage Changes

### PR DESCRIPTION
**Pull request by jbalsas**
Hey @marcoscv-work, @wincent, this is a quick, dirty, hacky change to see how what we're discussing over at https://docs.google.com/document/d/13K-jishFtNLOoWCsWffjsCc6xsfaiH9_-ofXwoGdx3Y/edit#heading=h.80rsx7we98gd actually looks like.

Check out [LPS-111253 Usage Changes](https://github.com/jbalsas/liferay-portal/pull/2166/commits/7487e19a8c3ab6637090d89c9faada80a44eed98) to see how a specific view will be transformed with this approach.

Check out [LPS-111253 Tag Changes](https://github.com/jbalsas/liferay-portal/pull/2166/commits/5f11173cbbba32ca1195ea4701dc7dce2af65679) to see changes in the tag implementation.

One "magic" aspect here to simplify the usage is that both `Row` and `Col` will try to look for an `autofit` parent to default to `autofit` even if an explicit value for it is not set. Not 100% this is a good idea, just trying it out here...

### CODE REVIEW

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content.jsp [hunk 1]
```diff
@@ -54,36 +61,48 @@ BlogsPortletInstanceConfiguration blogsPortletInstanceConfiguration = BlogsPortl
...
+				</clay:col>
 
-				<div class="autofit-col autofit-col-expand">
-					<div class="autofit-row">
-						<div class="autofit-col autofit-col-expand">
+				<clay:col
+					autofit="expand"
+				>
+					<clay:row
+						autofit="row"
```
Comment (jbalsas) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content.jsp:101]: Oops, this is not needed

**Comment by wincent** [comment]
I don't have an opinion about the ergonomics of this one — I'd rather leave that to the experts like @marcoscv-work — but my only technical doubt is whether there are any performance costs related to `findAncestorWithClass` that could make this prohibitive (or not!). What are your thoughts on that?

**Comment by jbalsas** [comment]
> [...] my only technical doubt is whether there are any performance costs related to findAncestorWithClass that could make this prohibitive (or not!). What are your thoughts on that?

There is indeed a performance penalty to pay there... being a layout utility, it's very likely this will show up when profiling live sites if applied across the board... that's why the idea of a different set of tags proposed in https://github.com/liferay/clay/pull/3204#issuecomment-628444321 is starting to look more appealing to me...

**Comment by jbalsas** [comment]
Superseeded by https://github.com/wincent/liferay-portal/pull/270

**Pull request closed without merging**